### PR TITLE
bumping both mocha versions

### PR DESCRIPTION
**Pull request by bfitzpat**
Signed-off-by: bfitzpat@redhat.com <bfitzpat@redhat.com>

### CODE REVIEW

**Comment by bfitzpat** [comment]
One adjustment to the mocha runner got us around the failure in https://github.com/camel-tooling/vscode-camelk/pull/476

File: src/test/suite/index.ts [hunk 1]
```diff
@@ -36,15 +36,14 @@ function loadCoverageRunner(testsRoot: string): CoverageRunner | undefined {
 	}
 }
 
-let mocha = new Mocha({
+// Create the mocha test
+const mocha = new Mocha({
 	ui: 'tdd',
 	timeout: 20000,
 	reporter: 'mocha-jenkins-reporter',
-	useColors: true
```
Comment (lhein) [inline src/test/suite/index.ts:43]: regarding https://github.com/mochajs/mocha/pull/4178 we can still use `colors` instead of `useColors`.

Comment (bfitzpat) [reply src/test/suite/index.ts:43]: good to know. I wasn't sure what to do with it, but useColors definitely threw an error

Comment (bfitzpat) [reply src/test/suite/index.ts:43]: looks like it's color instead of colors, but I'm trying it here

Comment (bfitzpat) [reply src/test/suite/index.ts:43]: yup. that worked.

**Pull request merged**